src/app.ts

```diff
@@ -195,6 +195,45 @@ export function build(opts = {}) {
     }
   )
 
+  server.post(
+    '/request/credential', async (request, reply) => {
+      const requestInfo: any = request.body;
+      const verifiablePresentation = requestInfo.verifiablePresentation;
+      // provided by issuer via diploma, email, LMS (e.g., Canvas), etc.
+      const challenge = verifiablePresentation?.proof?.challenge;
+      // retrieved from issuer query parameter in first URL delivered
+      // by issuer via diploma, email, LMS (e.g., Canvas), etc.
+      const verificationMethod = requestInfo.verificationMethod;
```

Comment:
Should we store this in an environment variable in the issuer deployment environment or should we have the wallet fetch it from the OpenID Provider Configuration, as suggested in my comment above?